lib/v2/blocs/deeplink/usecase/get_signing_request_use_case.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:async/async.dart';
+
+// ignore: import_of_legacy_library_into_null_safe
+import 'package:seeds/v2/datasource/local/settings_storage.dart';
+import 'package:seeds/v2/datasource/local/util/seeds_esr.dart';
+
+class GetSigningRequestUseCase {
```

Comment:
pls leave a comment on this use case. Its hard know why or how its used by just reading it. and the error message doesnt help understand

Reply:
I dislike comments but OK